tests/ProxyManagerTest/Functional/LazyLoadingGhostFunctionalTest.php

```diff
@@ -829,4 +829,70 @@ public function getPropertyAccessProxies()
             ],
         ];
     }
+
+    /**
+     * @dataProvider skipPropertiesFixture
+     *
+     * @param string   $className
+     * @param string   $propertyName
+     * @param string   $expected
+     * @param string[] $properties
+     */
+    public function testSkipProperties($className, $propertyName, $expected, $properties)
+    {
+        $proxy = $this->generateProxy($className);
+        $ghostObject = new $proxy(
+            function () use ($propertyName) {
+                $this->fail(sprintf('The Property "%s" was not expected to be lazy-loaded', $propertyName));
+            },
+            $properties
+        );
+
+        $property = new \ReflectionProperty($className, $propertyName);
+        $property->setAccessible(true);
+
+        $this->assertSame($expected, $property->getValue($ghostObject));
+    }
+
+    public function skipPropertiesFixture()
+    {
+        return [
+            [
+                ClassWithPublicProperties::class,
+                'property9',
+                'property9',
+                [
+                    'skippedProperties' => ["property9"]
+                ]
+            ],
+            [
+                ClassWithProtectedProperties::class,
+                'property9',
+                'property9',
+                [
+                    'skippedProperties' => ["\0*\0property9"]
+                ]
+            ],
+            [
+                ClassWithPrivateProperties::class,
+                'property9',
+                'property9',
+                [
+                    'skippedProperties' => [
+                        "\0ProxyManagerTestAsset\\ClassWithPrivateProperties\0property9"
+                    ]
+                ]
+            ],
+            [
+                ClassWithCollidingPrivateInheritedProperties::class,
```

Comment:
Can you add one test with the same class, but filtering out the parent class' property?